2-mandatory/8-journey-planner.js

```diff
@@ -19,8 +19,15 @@
   What's better about each approach?
 */
 
-function journeyPlanner(locations, transportMode) {}
-
+function journeyPlanner(locations, transportMode) {
+  const locationsByMYTransportMode = [];
+  for (const location in locations) {
+    let arrayTransport = locations[location];
+    if (arrayTransport.includes(transportMode) === true)
```

Comment:
```suggestion
    if (arrayTransport.includes(transportMode) )
```
An alternative to use check if true without the '=== true'

Reply:
Dear @SaadiaELF Thank you so much for of all your feedbacks. they are all really very useful.